dub.json

```diff
@@ -8,7 +8,8 @@
 	],
 	"dependencies": {
   		"vibe-d": "~>0.7.22",
-		"libdparse": {"optional": true, "version": "~>0.2.0"}
+		"libdparse": {"optional": true, "version": "~>0.2.0"},
+		"hyphenate": "~>1.1.0"
```

Comment:
It's just a single module and a pattern file, so I thought it wasn't worth the effort to handle `Have_hyphenate` everywhere (in particular b/c it's a command line option).